vendor/github.com/venicegeo/geojson-go/geojson/bbox.go

```diff
@@ -211,6 +211,12 @@ func (bb BoundingBox) Valid() error {
 	case 0:
 		return nil
 	case 4:
+		if bb[0] < -180 || bb[0] > 180 || bb[2] < -180 || bb[2] > 180 {
+			return errors.New("Longitude values must be between -180 and 180")
+		}
+		if bb[1] < -90 || bb[1] > 90 || bb[3] < -90 || bb[3] > 90 {
+			return errors.New("Latitude values must be between -90 and 90")
+		}
```

Comment:
Two issues here.
1. Since the GeoJSON library is vendored in, we need to fix the library at its source ([github.com/venicegeo/geojson-go](https://github.com/venicegeo/geojson-go)), not here.
2. GeoJSON does not mandate that coordinate values are lon/lat. They could be in another coordinate reference system that is not constrained to +/- 180, +/- 90. Therefore we will need an independent function to test the coordinate values. I propose a function called `ValidateLonLatCoordinates` in _geometry.go_ that takes `[]float64`. Better would be a function that takes `interface{}`, determines what type the input is, and reenters itself as needed to drill into multi-dimensional arrays. See `NewBoundingBox()` as an example for how this is done.